src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import styled from "@emotion/styled";
+import ExpandableCard from "@leafygreen-ui/expandable-card";
+import { Body } from "@leafygreen-ui/typography";
+
+import { Field } from "@rjsf/core";
+import { size } from "constants/tokens";
+import { triggers } from "constants/triggers";
+import { Subscriber } from "gql/generated/types";
+import { ResourceType, Trigger } from "types/triggers";
+import { SubscriptionSelect } from "./NotificationsRow";
+
+const getSentanceCase = (string: string) =>
+  string[0].toUpperCase() + string.substring(1).toLowerCase();
+
+export const getSubscriberType = (subscriber: Subscriber) => {
+  if (subscriber === undefined) {
+    return "";
+  }
+  let key: string = "";
+  Object.keys(subscriber).forEach((k) => {
+    if (subscriber[k] !== null) {
+      key = k;
+      return k;
+    }
+    return null;
+  });
+  return key;
+};
+
+export const getTrigger = (trigger, resourceType: string) => {
+  let triggerData: Trigger = {
+    trigger: "",
+    resourceType: ResourceType.VERSION,
+    regexSelectors: [],
+    label: "",
+  };
+  if (trigger === undefined) {
+    return triggerData;
+  }
+  triggers.forEach((t) => {
+    console.log("trigger: ", JSON.stringify(trigger));
+    if (t.trigger === trigger && t.resourceType === resourceType) {
+      triggerData = t;
+      return t;
+    }
+  });
+  return triggerData;
+};
+
+export const SubscriptionField: Field = ({ formData }) => {
+  const {
+    resourceType,
+    trigger,
+    selectors,
+    regexSelectors,
+    subscriber,
+  } = formData;
+
+  const subscriberType = getSubscriberType(subscriber) || "";
+  const subscriberName = subscriber ? subscriber[subscriberType] : "";
+  const triggerData = getTrigger(trigger, resourceType);
+
+  const title =
+    resourceType && trigger && subscriberName
+      ? `${getSentanceCase(resourceType)} ${trigger} ${subscriberName}`
+      : "New Subscription";
+
+  return (
+    <StyledExpandableCard
+      defaultOpen
+      contentClassName="patch-alias-card-content"
+      title={
+        <>
+          <TitleWrapper data-cy="expandable-card-title">{title}</TitleWrapper>
+        </>
+      }
+    >
+      <Body>
+        <SubscriptionSelect
+          resourceType={resourceType}
+          trigger={triggerData}
+          selectors={selectors}
+          regexSelectors={regexSelectors}
+        />
+        <br />
+        {/* todo: display properly */}
+        <strong>subscriberType: </strong> {getSubscriberType(subscriber)} <br />
```

Comment:
Instead of calling `getSubscriberType` again, you could use the variable `subscriberType` you defined above!